src/PAModel/SourceTransforms/AppTestTransform.cs

```diff
@@ -31,7 +31,7 @@ private class TestStepsMetadataJson
 
         public static bool IsTestSuite(string templateName)
         {
-            return templateName == "AppTest";
+            return templateName == "TestSuite";
```

Comment:
When read control nodes to be added into app._screens collection we need to check if it is of type TestSuite then we should't persist as a parent node, since the parent would be AppTest.
Earlier we used to use the parent control (AppTest) to source test file, now we will shard each test suite to its individual file